src/frame.js

```diff
@@ -0,0 +1,48 @@
+class Frame {
+
+  constructor(roll) {
+    this.frame = {
```

Comment:
I might not be familiar enough with the code yet to see the reasoning, but would it be clearer to have 2 properties defined in the Frame constructor, instead of combining rolls and bonus into one using an object? It might make the code more readable (e.g. frameInstance.roll[1] instead of current: frameInstance.frame.roll[1]).